core/src/main/java/org/apache/iceberg/BaseTable.java

```diff
@@ -232,7 +232,7 @@ public ExpireSnapshots expireSnapshots() {
 
   @Override
   public ManageSnapshots manageSnapshots() {
-    return new SnapshotManager(name, ops);
+    return new SnapshotManager(name, ops, reporter);
```

Comment:
can you please add some tests for this? Those tests can be similar to what we have in `ScanPlanningAndReportingTestBase`